plugins/inputs/beat/beat.go

```diff
@@ -0,0 +1,278 @@
+package beat
+
+import (
+	"encoding/json"
+	"net/http"
+	"net/url"
+	"time"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/internal"
+	"github.com/influxdata/telegraf/internal/tls"
```

Comment:
This is 
```suggestion
	"github.com/influxdata/telegraf/plugins/common/tls"
```
now I think.

Reply:
will fix